syserol/dataImport.py

```diff
@@ -122,3 +122,16 @@ def createCube():
     assert ~np.any(np.all(np.isnan(cube), axis=(1, 2)))
 
     return cube, glyCube
+
+def funcRegImport():
+    df = importLuminex()
+    
+    df1 = df[df["variable"].str.contains("IgG1")].pivot(index="subject", columns="variable")
+    df2 = df[df["variable"].str.contains("IgG2")].pivot(index="subject", columns="variable")
```

Comment:
Why do you have to separately pivot these?

Reply:
Robby wrote this function so I'm not entirely sure. I think there were issues and this was the way he got it to work? I can request his review.